website/static/js/contribAdder.js

```diff
@@ -313,13 +336,39 @@ var AddContributorViewModel = oop.extend(Paginator, {
         }
         return false;
     },
-    submit: function() {
+    selectAllNodes: function () {
+        //select all nodes to add a contributor to.  THe changed variable is set here for timing between
+        // treebeard and knockout
+        var self = this;
+        var nodesState = ko.toJS(self.nodesState());
+        for (var key in nodesState) {
+            if (nodesState[key].enabled) {
+                nodesState[key].checked = true;
+                nodesState[key].changed = nodesState[key].checked !== self.nodesOriginal[key].checked;
```

Comment:
Is it necessary to set `changed` here, given that the subscriber will do that here: https://github.com/CenterForOpenScience/osf.io/pull/4958/files#diff-8bccb237dc02931ca4e0314ade877143R54 ?